### PR TITLE
test upgrading from rook 1.0 to 1.5 in an airgapped environment

### PR DESCRIPTION
**Pull request by laverya**
#### What this PR does / why we need it:

This tests the rook 1.0 to 1.4 migration script in airgapped mode.

It's currently running as https://testgrid.kurl.sh/run/laverya%20rook%201.0%20to%201.5%20airgap%204%20(real).

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: testgrid/specs/full.yaml [hunk 1]
```diff
@@ -1681,6 +1681,62 @@
...
+  airgap: true
+  installerSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: "latest"
+    weave:
+      version: "latest"
+    rook:
+      version: "latest"
```
Comment (rrpolanco) [inline testgrid/specs/full.yaml:1697]: I think we should be explicit with the rook version. In this case `latest` = `1.0.4`

Comment (laverya) [reply testgrid/specs/full.yaml:1697]: good point, will change

File: testgrid/specs/full.yaml [hunk 2]
```diff
@@ -1681,6 +1681,62 @@
...
+  upgradeSpec:
+    kubernetes:
+      version: 1.20.x
+    containerd:
+      version: "latest"
+    weave:
+      version: "latest"
+    rook:
+      version: "1.5.12"
```
[the remark this answers is not in the document]
Comment (laverya) [reply testgrid/specs/full.yaml:1710]: I'll change the name instead 😄

**Comment by laverya** [comment]
just saw this log line:
```
2022-09-13 15:47:16+00:00 Please move this file to assets/rookupgrade-10to14.tar.gz before rerunning the installer.
```

That needs a bit of improvement (it's `/var/lib/kurl/assets/...`, but a user would not know that)

**Review by rrpolanco — DISMISSED** [review]

**Review by rrpolanco — APPROVED** [review]

**Comment by laverya** [comment]
The migration is currently failing with the error [here](https://testgrid.kurl.sh/run/laverya%20rook%201.0%20to%201.5%20airgap%209%20better%20status%20print%20staging?kurlLogsInstanceId=qrjwwbzimnjilmoo&nodeId=qrjwwbzimnjilmoo-initialprimary#L4455): 
```
Error: rook failed to become healthy within a minute, aborting migration: timed out waiting rook to become healthy, currently "health is HEALTH_WARN because \"7 pool(s) have non-power-of-two pg_num\""
```

Trying to work out why

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**